_commands/basics/id.md

```diff
@@ -16,9 +17,33 @@ ToDo: Needs completion
 
 #### Example command
 
-##### Break it down
+to find a specific user's UID, add -u {username}. Moreover, id -G USERNAME will find all the groups a user belongs to. Force id to display the name of the UID or GID instead of ugly numbers by passing the -n options
+
+~~~ bash
+$id -ng username
+staff eecsstudents club1 club2
+~~~
 
 #### Example command
+To find all the groups associated with a user called exampleuser, enter:
+
+~~~ bash
+$ id exampleuser
+~~~
+
+sample output:
+
+~~~ bash
+uid = 0(exampleuser) gid=0(eecs) groups=1(computing), 1(hello),2(this), 3(is), 4(an), 5(example)
+~~~
 
-##### Break it down
+### Useful options:
```

Comment:
That's embarrassing...I should be done, then!

On Sun, Mar 27, 2016 at 9:01 PM, Waleed Khan <notifications@github.com>
wrote:

> In _commands/basics/id.md
> <https://github.com/c4cs/c4cs.github.io/pull/161#discussion_r57537992>:
>
> >
> > -##### Break it down
> > +### Useful options:
>
> You should just be able to push your changes to your Github fork and the
> PR should update automatically, no?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/c4cs/c4cs.github.io/pull/161/files/e21f4daf2cd9db1d3aabf6c38fcd78d5c578f67f#r57537992>
>